common/openapi.yml

```diff
@@ -361,12 +361,14 @@ components:
       description: |
         Object containing the user-provided information about a job request.
       type: object
-      required: [ pipelineVersion, pipelineInputs ]
+      required: [ jobControl, pipelineVersion, pipelineInputs ]
       properties:
         description:
           description: >-
             User-provided description of the job request.
           type: string
+        jobControl:
```

Comment:
just curious, what is the benefit of this schema being an object vs a string like pipelineVersion?

Reply:
it's the spec. it's an object that contains currently just the "id" field, but in future it could additionally contain things like notification preferences. does this answer your question?

Reply:
👍